### PR TITLE
create a base class for the pages

### PR DESCRIPTION
**Pull request by mariasergeenko**

### CODE REVIEW

File: tests/e2e/pageObjects/base-run-commands-page.ts [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+    getTutorialLinkLocator = (tutorialName: string): Selector =>
+        Selector(`[@data-testid="data-testid=query-tutorials-link_${tutorialName}"]`);
+
+    /**
+     * Get card container by command
+     * @param command The command
+     */
+    async getCardContainerByCommand(command: string): Promise<Selector> {
+        return this.queryCardCommand.withExactText(command).parent('[data-testid^="query-card-container-"]');
```
Comment (vlad-dargel) [inline tests/e2e/pageObjects/base-run-commands-page.ts:40]: I know that you just moved this from workbench-page
but lets store '[data-testid^="query-card-container-"]' separately in css selectors above

File: tests/e2e/pageObjects/base-run-commands-page.ts [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+        // Compare the command result with executed command
+        const actualCommandResult = await this.queryCardContainer.nth(childNum).find(this.cssQueryTextResult).textContent;
+        await t.expect(actualCommandResult).contains(result, 'Actual command result is not equal to executed');
+    }
+
+    /**
+     * Get selector with tutorial name
+     * @param tutorialName name of the uploaded tutorial
+     */
+    getAccordionButtonWithName(tutorialName: string): Selector {
```
Comment (vlad-dargel) [inline tests/e2e/pageObjects/base-run-commands-page.ts:75]: lets store this method as Selector like getTutorialLinkLocator above

File: tests/e2e/pageObjects/base-run-commands-page.ts (hunk 1 above)
Comment (mariasergeenko) [reply tests/e2e/pageObjects/base-run-commands-page.ts:40]: fixed

File: tests/e2e/pageObjects/base-run-commands-page.ts (hunk 2 above)
Comment (mariasergeenko) [reply tests/e2e/pageObjects/base-run-commands-page.ts:75]: remove the selector since it is not used

**Review by vlad-dargel — APPROVED** [review]

**Pull request merged**